additions, clarifications, markup (#45)
neighbourhoodie/stf-php-ansible: README.md

[Here are some tips for making the most of Ansible and Ansible playbooks.](https://docs.ansible.com/ansible/2.8/user_guide/playbooks_best_practices.html#best-practices)

## General information

System setup is done in two phases. Phase 1 uses ssh connections backed by SSH keys for the `root` user on each machine. In our case these are provisioned by DigitalOcean on droplet creation. In this phase, admin accounts with their own SSH keys as well as Google Authenticator are set up. In Phase 2, the accounts and 2FA set up in Phase 1 are used to connect to all machines.

## Prerequisite

- Ansible configuration: Before running the initialization script, comment out the `ssh_connection` setting in the ansible.cfg file. This ensures the initialization process is performed using the root user.
- Update jumphost domains: Modify domains in `/bin/auth-jump0-1` and `/etc/ssh_config_jump0-1`.

> - Create a vault password: Before starting, you must create a vault password. This password can be anything you choose. Refer to the [Ansible Vault Guide](https://docs.ansible.com/ansible/2.8/user_guide/vault.html). Run the following to change the password. You’ll be prompted to enter the old password and the new password. Then update the variable values as required. The password for the placeholder secrets in this repo is `123`. These are not used in any production systems anywhere.

Reviewer: We need to check. @ninetteadhikari ?

> - Create a vault password: Before starting, you must create a vault password. This password can be anything you choose. Refer to the [Ansible Vault Guide](https://docs.ansible.com/ansible/2.8/user_guide/vault.html). Run the following to change the password. You’ll be prompted to enter the old password and the new password. Then update the variable values as required. The password for the placeholder secrets in this repo is `123`. These are not used in any production systems anywhere.

Reviewer: ```suggestion
- Create a vault password: Before starting, you must create a vault password. This password can be anything you choose. Refer to the [Ansible Vault Guide](https://docs.ansible.com/ansible/2.8/user_guide/vault.html). Run the following to change the password. You’ll be prompted to enter the old password and the new password. Then update the variable values as required. The password for the placeholder secrets in this repo will be shared seperately. These are not used in any production systems anywhere.
```
